scripts/yggdrasil-iptunnel/install

```diff
@@ -6,6 +6,7 @@ BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 # Dependencies
 sudo apt-get -y install confget
+sudo cp -f ../confset.sh /usr/local/bin/confset.sh
```

Comment:
This should be part of the generic install script, if you're going to leave the file as `scripts/confset.sh`. If you want to make it a module I'm fine with that too, but the way I'm suggesting seems easier.

Reply:
agreaed

#389